addons/stock_picking_batch/tests/test_wave_picking.py

```diff
@@ -119,8 +120,37 @@ def setUpClass(cls):
             'location_id': cls.stock_location.id,
             'location_dest_id': cls.customer_location.id,
         })
+
+        cls.picking_client_4 = cls.env['stock.picking'].create({
```

Comment:
Maybe specify that this one is internal in the name so it's more obviously different from the other 3? Might need to update the docstring too since that describes the pickings it's creating.

Reply:
Done